aws_dynamodb_table resource can not be created in cn-north-1 region

I try creating the following resource in cn-north-1 region:
```
resource "aws_dynamodb_table" "dynamodb-terraform-state-lock" {
  name = "${var.dynamo_lock_table_name}"
  hash_key = "LockID"
  read_capacity = 20
  write_capacity = 20

  point_in_time_recovery {
    enabled = false
  }

  attribute {
    name = "LockID"
    type = "S"
  }
}
```
The table is created, but terraform script fails with an error that dynamodb/DescribeContinuousBackups is not available.
```
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: ---[ REQUEST POST-SIGN ]-----------------------------
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: POST / HTTP/1.1
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: Host: dynamodb.cn-north-1.amazonaws.com.cn
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: User-Agent: aws-sdk-go/1.13.47 (go1.9.2; linux; amd64) APN/1.0 HashiCorp/1.0 Terraform/0.11.7
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: Content-Length: 33
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: Accept-Encoding: identity
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: Authorization: AWS4-HMAC-SHA256 Credential=AKIAPQ7ALHIM46ZI5XCA/20180517/cn-north-1/dynamodb/aws4_request, SignedHeaders=accept-encoding;content-length;content-type;host;x-amz-date;x-amz-target, Signature=e96718eaeb461b5d6a53d91a80e605745724e2f802b6121b9b81f5350cc37fb7
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: Content-Type: application/x-amz-json-1.0
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: X-Amz-Date: 20180517T143853Z
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: X-Amz-Target: DynamoDB_20120810.DescribeContinuousBackups
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: 
2018-05-17T14:38:53.863Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: -----------------------------------------------------
module.dynamo.aws_dynamodb_table.dynamodb-terraform-state-lock: Still creating... (20s elapsed)
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: 2018/05/17 14:38:55 [DEBUG] [aws-sdk-go] DEBUG: Response dynamodb/DescribeContinuousBackups Details:
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: ---[ RESPONSE ]--------------------------------------
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: HTTP/1.1 400 Bad Request
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: Connection: close
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: Content-Length: 103
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: Content-Type: application/x-amz-json-1.0
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: Date: Thu, 17 May 2018 14:38:54 GMT
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: Server: Server
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: X-Amz-Crc32: 2885926038
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: X-Amzn-Requestid: UFLP3UOJ40DCHIAN9KD3RU68A3VV4KQNSO5AEMVJF66Q9ASUAAJG
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: 
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: 
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: -----------------------------------------------------
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: 2018/05/17 14:38:55 [DEBUG] [aws-sdk-go] {"__type":"com.amazon.coral.service#UnknownOperationException","message":"Unknown operation exception"}
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4: 2018/05/17 14:38:55 [DEBUG] [aws-sdk-go] DEBUG: Validate Response dynamodb/DescribeContinuousBackups failed, not retrying, error UnknownOperationException: Unknown operation exception
2018-05-17T14:38:55.803Z [DEBUG] plugin.terraform-provider-aws_v1.19.0_x4:      status code: 400, request id: UFLP3UOJ40DCHIAN9KD3RU68A3VV4KQNSO5AEMVJF66Q9ASUAAJG
2018/05/17 14:38:55 [ERROR] root.dynamo: eval: *terraform.EvalApplyPost, err: 1 error(s) occurred:

* aws_dynamodb_table.dynamodb-terraform-state-lock: UnknownOperationException: Unknown operation exception
        status code: 400, request id: UFLP3UOJ40DCHIAN9KD3RU68A3VV4KQNSO5AEMVJF66Q9ASUAAJG
2018/05/17 14:38:55 [ERROR] root.dynamo: eval: *terraform.EvalSequence, err: 1 error(s) occurred:

* aws_dynamodb_table.dynamodb-terraform-state-lock: UnknownOperationException: Unknown operation exception
        status code: 400, request id: UFLP3UOJ40DCHIAN9KD3RU68A3VV4KQNSO5AEMVJF66Q9ASUAAJG

```

--- a/aws/resource_aws_dynamodb_table.go
+++ b/aws/resource_aws_dynamodb_table.go
@@ -527,7 +527,7 @@ func resourceAwsDynamoDbTableRead(d *schema.ResourceData, meta interface{}) erro
 	pitrOut, err := conn.DescribeContinuousBackups(&dynamodb.DescribeContinuousBackupsInput{
 		TableName: aws.String(d.Id()),
 	})
-	if err != nil {
+	if err != nil && !isAWSErr(err, "UnknownOperationException", "") {
 		return err
 	}
 	d.Set("point_in_time_recovery", flattenDynamoDbPitr(pitrOut))

--- a/aws/structure.go
+++ b/aws/structure.go
@@ -3697,15 +3697,19 @@ func flattenDynamoDbTtl(ttlDesc *dynamodb.TimeToLiveDescription) []interface{} {
 }
 
 func flattenDynamoDbPitr(pitrDesc *dynamodb.DescribeContinuousBackupsOutput) []interface{} {
-	m := map[string]interface{}{}
+	m := map[string]interface{}{
+		"enabled": false,
+	}
+
+	if pitrDesc == nil {
+		return []interface{}{m}
+	}
+
 	if pitrDesc.ContinuousBackupsDescription != nil {
 		pitr := pitrDesc.ContinuousBackupsDescription.PointInTimeRecoveryDescription
 		if pitr != nil {
 			m["enabled"] = (*pitr.PointInTimeRecoveryStatus == dynamodb.PointInTimeRecoveryStatusEnabled)
 		}
-	}
-	if len(m) > 0 {
-		return []interface{}{m}
 	}
 
 	return []interface{}{m}

